lettuce/ext/_reporter/vtk_reporter.py

```diff
@@ -4,7 +4,7 @@
 
 from ... import Reporter
 
-__all__ = ['VTKReporter']
+__all__ = ['VTKReporter', 'write_vtk']
```

Comment:
Should we make this a public function? Why not keep it guarded behind `VTKReporter`?